sdk/examples/client/output/nft.rs

```diff
@@ -26,12 +29,11 @@ async fn main() -> Result<()> {
     // non-zero balance.
     dotenvy::dotenv().ok();
 
-    let node_url = std::env::var("NODE_URL").unwrap();
-    let explorer_url = std::env::var("EXPLORER_URL").unwrap();
-    let faucet_url = std::env::var("FAUCET_URL").unwrap();
```

Comment:
I would argue it was better to do it only once but 🤷🏻

Reply:
Performance-wise it may be better, but the focus in the examples IMHO is the SDK functionality, not reading env vars.

Reply:
Well we end up reading more env vars now, so I'd say you're putting more focus now than before but ok

Reply:
Changed it back.